### PR TITLE
New package: stcgal-1.6

### PR DESCRIPTION
**Pull request by bra1nwave**

### CODE REVIEW

File: srcpkgs/stcgal/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+depends="python3-pyserial python3-setuptools python3-tqdm"
+short_desc="STC MCU ISP flash tool"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="MIT"
+homepage="https://github.com/grigorig/stcgal"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=1a526503226886a6c4d7fb8296e5530053bcac63fe6f4a29d5f6c40f86f9e20f
+
+post_install() {
+	vlicense ${FILESDIR}/LICENSE
```
Comment (maxice8) [inline srcpkgs/stcgal/template]: i'm not keen on vendoring the license, can't you put the link on the distfiles ?

Comment (bra1nwave) [reply srcpkgs/stcgal/template]: Sadly, there is no license file in that repo. 
I could take the license from the [AUR](https://aur.archlinux.org/cgit/aur.git/plain/LICENSE?h=stcgal) (if that's allowed, of course, which I doubt).

Comment (Johnnynator) [reply srcpkgs/stcgal/template]: @bra1nwave Can't you extract the License out of the Source files? Like `head -n21 stcgal/frontend.py > LICENSE`

Comment (bra1nwave) [reply srcpkgs/stcgal/template]: Good idea, Johnnynator. Thanks.

File: srcpkgs/python-tqdm/template [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="MPL-2.0, MIT"
+homepage="https://tqdm.github.io/"
+distfiles="https://github.com/tqdm/tqdm/archive/v${version}.tar.gz"
+checksum=3a405bcf0b4f30c72d2f7d10f7574d952b837abb5777dc2ed0f0a57a8943d171
+alternatives="
+ tqdm:tqdm:/usr/bin/tqdm2
+ tqdm:tqdm.1:/usr/share/man/man1/tqdm2.1"
+
+post_install() {
```
Comment (maxice8) [inline srcpkgs/python-tqdm/template]: There is a manpage in ${py3_sitelib} and ${py2_sitelib}

Comment (bra1nwave) [reply srcpkgs/python-tqdm/template]: Not anymore

File: srcpkgs/python-tqdm/template [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+python3-tqdm_package() {
+	alternatives="
+	 tqdm:tqdm:/usr/bin/tqdm3
+	 tqdm:tqdm.1:/usr/share/man/man1/tqdm3.1"
+	noarch=yes
+	pycompile_module="tqdm"
+	depends="python3"
+	short_desc="${short_desc/2/3}"
```
Comment (maxice8) [inline srcpkgs/python-tqdm/template]: please replace Python2/Python3

**Pull request closed without merging**